public_html/wp-content/mu-plugins/blocks/assets/src/blocks.js

```diff
@@ -10,6 +10,7 @@ import * as organizers from './organizers/';
 import * as sessions from './sessions/';
 import * as speakers from './speakers/';
 import * as sponsors from './sponsors/';
+import './blocks-store';
```

Comment:
Do we need this here? It doesn't look like anything in this file is using the store...

Reply:
Removed in b556a62a3de2f46d3ff570f48d9649585d9c78fc